deployments/namespaces.yaml

```diff
@@ -2,4 +2,9 @@ apiVersion: v1
 kind: Namespace
 metadata:
    name: admin
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+   name: monitoring
```

Comment:
will this causes issues because its already defined in flux?

Reply:
It shouldn't because the manifest should just synchronize. Tested in Perftest01. But I think I will remove it from flux config.